### PR TITLE
ch16/ssh-tunnel

### PR DESCRIPTION
**Pull request by dalygbarron**
In this test it just always tunnels to some server and then connects and saves some inane junk about kittens.
you will need these variables in api/.env.dev.yml
```
SOSURE_MONGO_SSH_HOST: web-1.staging.so-sure.net
SOSURE_MONGO_SSH_USER: daly
SOSURE_MONGO_HOST: 127.0.0.1
SOSURE_MONGO_DB: so-sure
SOSURE_MONGO_USER: root
SOSURE_MONGO_PASSWORD: root
```

### CODE REVIEW

File: api/tests/mongo.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    mongoose.connect(
+        'mongodb://localhost:27017/' + process.env.SOSURE_MONGO_DB
+    );
+    let db = mongoose.connection;
+    db.on('error', console.error.bind(console, 'connection error:'));
+    db.once('open', function () {
+        let tango = new Kitten({name: "Tango", alive: true});
+        let bongo = new Kitten({name: "Bongo", alive: true});
+        tango.attack(bongo);
+        tango.save(function (error, tango) {
```
Comment (elliotmitchell) [inline api/tests/mongo.js:50]: Is this actually making DB changes?

Comment (elliotmitchell) [reply api/tests/mongo.js:50]: It's fine if so, we'll just get it in and I'll make some changes

Comment (dalygbarron) [reply api/tests/mongo.js:50]: yes

Comment (elliotmitchell) [reply api/tests/mongo.js:50]: oh that's fine then

**Pull request merged**